Dockerfile

```diff
@@ -73,14 +73,13 @@ WORKDIR /ipxe
 FROM patch as precompile
 COPY etc /sample
 RUN make bin/undionly.kpxe
-RUN make bin/ipxe.iso
```

Comment:
This was not building with the new iPXE source, and because I did not see where this was used in this Dockerfile I yanked it to at least see my build through.

Here's the failure: https://jenkins.algol60.net/blue/rest/organizations/jenkins/pipelines/Cray-HPE/pipelines/ipxe-tpsw-clone/branches/MTL-2104/runs/6/nodes/81/steps/98/log/?start=0

Does anyone know if we need this?